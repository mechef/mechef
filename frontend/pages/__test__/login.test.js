import renderer from 'react-test-renderer';
import React from 'react';

import Login from '../login';

describe('Pages With Snapshot Testing', () => {
  it('snapshot login page', () => {
    const component = renderer.create(<Login />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
