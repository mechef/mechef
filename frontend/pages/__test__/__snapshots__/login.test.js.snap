// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages With Snapshot Testing snapshot login page 1`] = `
<div
  className="jsx-3294668015"
>
  <nav
    className="jsx-2255944320 mechef-navbar"
  >
    <img
      alt="logo"
      className="jsx-2255944320 mechef-navbar-logo"
      src="../static/food.png"
    />
    <div
      className="jsx-2255944320 mechef-navbar-collapse"
    >
      <ul
        className="jsx-2255944320 mechef-navbar-nav"
      >
        <li
          className="jsx-2255944320 "
        >
          HOW IT WORKS
        </li>
        <li
          className="jsx-2255944320 active"
        >
          JOIN NOW
        </li>
      </ul>
    </div>
  </nav>
  <div
    className="jsx-3294668015 login-panel"
  >
    <div
      className="jsx-3294668015 login-form"
    >
      <div
        className="jsx-3294668015 login-btn splits"
      >
        <p
          className="jsx-3294668015 splits-title"
        >
          Have an account?
        </p>
        <button
          className="jsx-675350272 
      button
      whiteBorderOnly
      small
    "
          onClick={[Function]}
          type="button"
        >
          SIGN IN
        </button>
      </div>
      <div
        className="jsx-3294668015 rgstr-btn splits"
      >
        <p
          className="jsx-3294668015 splits-title"
        >
          Dont have an account?
        </p>
        <button
          className="jsx-675350272 
      button
      whiteBorderOnly
      small
    "
          onClick={[Function]}
          type="button"
        >
          JOIN NOW
        </button>
      </div>
      <div
        className="jsx-3294668015 wrapper "
      >
        <div
          className="jsx-3294668015 login"
        >
          <p
            className="jsx-3294668015 title"
          >
            SIGN IN
          </p>
          <div
            className="jsx-3294668015 mail"
          >
            <div
              className="jsx-3090447383"
            >
              <input
                className="jsx-3090447383 
        textInput
        medium
      "
                onChange={[Function]}
                placeholder="Mail or Username"
                type="mail"
                value=""
              />
            </div>
          </div>
          <div
            className="jsx-3294668015 passwd"
          >
            <div
              className="jsx-3090447383"
            >
              <input
                className="jsx-3090447383 
        textInput
        medium
      "
                onChange={[Function]}
                placeholder="Password"
                type="password"
                value=""
              />
            </div>
          </div>
          <button
            className="jsx-675350272 
      button
      primary
      medium
    "
            onClick={[Function]}
            type="button"
          >
            SIGN IN
          </button>
        </div>
        <div
          className="jsx-3294668015 register"
        >
          <p
            className="jsx-3294668015 title"
          >
            Be a Chef today!
          </p>
          <div
            className="jsx-3294668015"
          >
            <div
              className="jsx-3090447383"
            >
              <input
                className="jsx-3090447383 
        textInput
        medium
      "
                onChange={[Function]}
                placeholder="First Name"
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            className="jsx-3294668015"
          >
            <div
              className="jsx-3090447383"
            >
              <input
                className="jsx-3090447383 
        textInput
        medium
      "
                onChange={[Function]}
                placeholder="Last Name"
                type="text"
                value=""
              />
            </div>
          </div>
          <div
            className="jsx-3294668015"
          >
            <div
              className="jsx-3090447383"
            >
              <input
                className="jsx-3090447383 
        textInput
        medium
      "
                onChange={[Function]}
                placeholder="Telephone Number"
                type="password"
                value=""
              />
            </div>
          </div>
          <div
            className="jsx-3294668015"
          >
            <div
              className="jsx-3090447383"
            >
              <input
                className="jsx-3090447383 
        textInput
        medium
      "
                onChange={[Function]}
                placeholder="Email Address"
                type="mail"
                value=""
              />
            </div>
          </div>
          <div
            className="jsx-3294668015 wrapper__submit"
          >
            <button
              className="jsx-675350272 
      button
      primary
      medium
    "
              onClick={[Function]}
              type="button"
            >
              JOIN NOW
            </button>
          </div>
          <div
            className="jsx-3294668015 wrapper__note"
          >
            <span
              className="jsx-3294668015"
            >
              If you click JOIN NOW, it means you agree with terms of service.
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
